applications/serial_lte_modem/doc/FOTA_AT_commands.rst

```diff
@@ -43,26 +45,67 @@ Syntax
 * The ``<apn>`` parameter is a string.
   It represents an alternative access point name (APN), other than the default primary APN, to use for downloading.
 
+Response syntax
+~~~~~~~~~~~~~~~
+
+::
+
+  #XFOTA: <primary_partition_id>,<MCUBOOT_version>,<img_size>,<img_version>
+  #XFOTA: <secondary_partition_id>,<MCUBOOT_version>,<img_size>,<img_version>
+
+* The ``<primary_partition_id>`` and ``<secondary_partition_id>`` integers that give the partition ID.
+* The ``<MCUBOOT_version>`` integer. The value 1 corresponds to MCUboot versions 1.x.y.
+* The ``<img_size>`` an integer that gives the size in bytes of the application image in this partition.
+* The ``<img_version>`` a string that gives the version of the application image as "<major>.<minor>.<revision>+<build_number>".
```

Comment:
I see versioning here is given as a string.

Reply:
This is version of the image given by "west sign" command, which is originally in the format of "x.y.x-b"